ruby/alias_manager.rb

```diff
@@ -0,0 +1,53 @@
+#PSEUDOCODE
+=begin
+- Create a method that takes in a name. 
+  -It splits that name into array of two, 
+  -It cycles through that array and for each word, substitutes vowels to the next vowel and consonants to the next consonant. 
+  -It then pushes that new word into first position of new_name array. Essentially, switching order. 
+  -It then makes that array into a string. 
+-end method
+-Make empty hash to store future names and corresponding aliases. 
+-begin loop that goes on until user types "quit".
+  -In loop, ask the user to type in a name. 
+    -call method in new_full_name variable.  
+    -Push new_full_name to names_and_aliases hash. 
+    -Print the new alias. 
+    -Create variable that stores the name originally entered capitalized so that it prints nicely later on.  
+  -end loop
+-print the names_and_aliases hash contents in list.      
+=end 
+
+def alias_maker (full_name)
+  
+  full = full_name.split(" ")
+  original_first_name = full[0]
+  original_last_name = full[1]
+  new_name = []
+
+  full.each_index do |x|
+    y = full[x].gsub(/[a-z]/, 
```

Comment:
@kimberlysuazo I like the gsub.  My first paying job post DBC was all regular expressions.

That said, you can and should totally do this algorithmically. This does work and is an acceptable answer but if you choose to refactor, consider finding a way to grab the next valid letter and conditionally check weather it is a vowel or not.  Pseudocode it out first and reach out on Slack or office hours tomorrow if you have any questions.